src/main/java/com/example/bookshop/model/OrderItem.java

```diff
@@ -0,0 +1,38 @@
+package com.example.bookshop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
+@Table(name = "order_items")
+public class OrderItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "order_id", nullable = false)
+    private Order order;
+    @OneToOne
+    @JoinColumn(name = "book_id", nullable = false)
+    private Book book;
+    @Column(nullable = false)
+    private int quantity;
```

Comment:
```suggestion
    private Integer quantity;
```
I prefer to use the wrapped type as you can tell if the object has been saved/loaded to/from the database whether or not the id value is null.